src/public/parts/shells/chat/public/src/templates/group/settings/ice_server_row.html

```
@@ -1,11 +1,8 @@
 <div class="grid grid-cols-1 md:grid-cols-4 gap-2 items-end" data-ice-row="1">
-	<input type="text" class="input input-bordered input-sm md:col-span-2" data-ice-url placeholder="stun:host:19302"
-		value="${urls}">
-	<input type="text" class="input input-bordered input-sm" data-ice-user placeholder="TURN user"
-		value="${username}">
+	<input type="text" class="input input-bordered input-sm md:col-span-2" data-ice-url placeholder="stun:host:19302" value="${urls}">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)